playbooks/hc-ansible-deployment/he_gluster_vars.json

```diff
@@ -13,5 +13,6 @@
   "he_bridge_if": "interface name for bridge creation",
   "he_enable_hc_gluster_service": true,
   "he_mem_size_MB": "16384",
-  "he_cluster": "Default"
+  "he_cluster": "Default",
+  "he_vcpus": 4
```

Comment:
I believe , you have mentioned 4 because the default value is 4, do you require a parameter so that customer can set the custom value as well, or is this enough?
@rchikatw

Reply:
This is enough.